### ISSUE TITLE
Document accessibility rules that the Slack Accessibility Bot will check

### ISSUE THREAD

**Issue by christinach** [pulibrary/slack-accessibility-bot#4]
### Introduction
We want to start testing specific rules. A document will help to drive this tests. 

### Acceptance criteria

- [ ] If there is no docs directory, create one.

- [ ] Update an existing markdown document or add a new one in the docs directory.

- [ ] It has introduction: Explains the goals and purpose of this documentation.

- [ ] It has a step by step guide: Provides the exact steps to guide the user on how to do a task.

#### Additional acceptance criteria 
- [ ] It includes references: References any related resources that have assisted creating this documentation (links to other tickets, online articles etc.).

- [ ] It includes workflow: Defines the steps in different environments (production, staging)
